index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mark Portfolio</title>
+</head>
+<body>
+    <header class="header-logo">
+        <h2><a href="#" class="LOGO">Mark</a></h2>
+        <nav>
```

Comment:
- [ ] Great work wrapping the logo in an anchor tag (`a`). However, it is not semantic to wrap the `a` tag in an `h2` tag. `hx` tags are used for headers or texts that should be significantly bigger than others. I would advise you to use a `div` tag instead of the `h2` tag.

Reply:
Used div tag